CONTRIBUTING.md

```diff
@@ -62,7 +61,9 @@ the pull request. This way, your change always consists of only one commit and
 can be merged in the instant everybody is content with the whole thing.
 
 
-[community repositories]: http://gluon.readthedocs.org/en/latest/user/site.html#packages
-[#gluon]: irc://irc.hackint.org/gluon
+[packages]: http://gluon.readthedocs.org/en/latest/user/site.html#packages
+[#gluon]: https://webirc.hackint.org/#gluon
```

Comment:
I'm not sure if we want to change this link to Webirc, as we occasionally have to set the channel to +r ...